gonum.go

```diff
@@ -0,0 +1,9 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
```

Comment:
I really don't want a "gonum.org/v1/gonum" to exist until it has a reason to exist for users.